src/model/regions_info.js

```diff
@@ -728,7 +727,13 @@ export default class RegionsInfo  {
      * @memberof RegionsInfo
      */
     resetShapeDefaults() {
-        this.shape_defaults['StrokeColor'] = -65281;
+        //if they set a color palette, set the first one as default stroke color
+        console.log(this.image_info.context.roi_color_palette)
```

Comment:
Don't need to console.log this all the time (although it was helpful for my debugging). OK to log it IF it's actually set.